nns/nns.yml

```diff
@@ -1,7 +1,7 @@
 name: "NameService"
 supportedstandards: ["NEP-11"]
 safemethods: ["balanceOf", "decimals", "symbol", "totalSupply", "tokensOf", "ownerOf",
-              "tokens", "properties", "roots", "getPrice", "isAvailable", "getRecord",
+              "tokens", "properties", "roots", "getPrice", "isAvailable", "getRecords",
```

Comment:
Hmm. I'd expect our compiler to catch missing `getRecord`.

Reply:
And it will. But I found out that `nns.yml` is unused config (there's `config.yml` instead), so I'll remove it.